iOS_SDK/OneSignalSDK/Source/OneSignalCommonDefines.h

```diff
@@ -30,56 +30,70 @@
 
 #import <Foundation/Foundation.h>
 
-// networking
+// Networking
 #define API_VERSION @"api/v1/"
 #define SERVER_URL @"https://onesignal.com/"
 
-// NSUserDefaults parameter names
-#define HAS_PROMPTED_FOR_NOTIFICATIONS @"OS_HAS_PROMPTED_FOR_NOTIFICATIONS"
-#define PROVISIONAL_AUTHORIZATION @"ONESIGNAL_PROVISIONAL_AUTHORIZATION"
-#define REGISTERED_WITH_APPLE @"GT_REGISTERED_WITH_APPLE"
-#define INAPP_LAUNCH_URL @"ONESIGNAL_INAPP_LAUNCH_URL"
-#define NOTIFICATION_PROMPT_ANSWERED @"OS_NOTIFICATION_PROMPT_ANSWERED"
-#define LAST_MESSAGE_OPENED @"GT_LAST_MESSAGE_OPENED_"
-#define CACHED_MEDIA @"CACHED_MEDIA"
-#define EMAIL_AUTH_CODE @"GT_EMAIL_AUTH_CODE"
-#define SUBSCRIPTION_SETTING @"ONESIGNAL_SUBSCRIPTION_LAST"
-#define EMAIL_USERID @"GT_EMAIL_PLAYER_ID"
-#define NSUD_APP_ID @"GT_APP_ID"
-#define USERID @"GT_PLAYER_ID"
-#define USERID_LAST @"GT_PLAYER_ID_LAST"
-#define USER_LAST_CLOSED_TIME @"GT_LAST_CLOSED_TIME"
-#define DEVICE_TOKEN @"GT_DEVICE_TOKEN"
-#define UNSENT_ACTIVE_TIME @"GT_UNSENT_ACTIVE_TIME"
-#define UNSENT_ACTIVE_TIME_ATTRIBUTED @"GT_UNSENT_ACTIVE_TIME_ATTRIBUTED"
-#define SUBSCRIPTION @"ONESIGNAL_SUBSCRIPTION"
-#define PUSH_TOKEN @"GT_DEVICE_TOKEN_LAST"
-#define ACCEPTED_PERMISSION @"ONESIGNAL_PERMISSION_ACCEPTED_LAST"
-#define REQUIRE_EMAIL_AUTH @"GT_REQUIRE_EMAIL_AUTH"
-#define EMAIL_ADDRESS @"EMAIL_ADDRESS"
-#define PROMPT_BEFORE_OPENING_PUSH_URL @"PROMPT_BEFORE_OPENING_PUSH_URL"
-#define DEPRECATED_SELECTORS @[@"application:didReceiveLocalNotification:", @"application:handleActionWithIdentifier:forLocalNotification:completionHandler:", @"application:handleActionWithIdentifier:forLocalNotification:withResponseInfo:completionHandler:"]
-#define USES_PROVISIONAL_AUTHORIZATION @"ONESIGNAL_USES_PROVISIONAL_PUSH_AUTHORIZATION"
-#define PERMISSION_HAS_PROMPTED @"OS_HAS_PROMPTED_FOR_NOTIFICATIONS_LAST"
-#define PERMISSION_ANSWERED_PROMPT @"OS_NOTIFICATION_PROMPT_ANSWERED_LAST"
-#define PERMISSION_ACCEPTED @"ONESIGNAL_ACCEPTED_NOTIFICATION_LAST"
-#define PERMISSION_PROVISIONAL_STATUS @"ONESIGNAL_PROVISIONAL_AUTHORIZATION_LAST"
-#define PERMISSION_PROVIDES_NOTIFICATION_SETTINGS @"OS_APP_PROVIDES_NOTIFICATION_SETTINGS"
-#define EXTERNAL_USER_ID @"OS_EXTERNAL_USER_ID"
+// OneSignalUserDefault keys
+// String values start with "OSUD_" to maintain a level of uniqueness fropm other apps
```

Comment:
"uniqueness fropm other apps" should be "uniqueness from other libraries or app code"

Reply:
Got it!